maud/data_model/maud_config.py

```diff
@@ -25,6 +26,7 @@ class MaudConfig:
     :param likelihood: Whether or not to take measurements into account.
     :param cmdstanpy_config: Arguments to cmdstanpy.CmdStanModel.sample.
     :param reject_non_steady: Reject draws if a non-steady state is encountered.
+    :param penalize_non_steady: Penalize the deviation from steady state in the log likelihood.
```

Comment:
To shut up ruff here and at line 38 just add `# noqa: F401`

Reply:
It should be fixed. Now that I have checked ruff, the fact that you can run `ruff --add-noqa file.py` is fantastic.